tutorials/classify-maizeland-ng/index.md

```diff
@@ -0,0 +1,220 @@
+
+```js
+// Ingest Sentinel-2A imageries
+var s2 = ee.ImageCollection('COPERNICUS/S2');
+
+// Define parameters for cloudmask; Bits 10 and 11 are clouds and cirrus, respectively.
+var cloudBitMask = ee.Number(2).pow(10).int();
+var cirrusBitMask = ee.Number(2).pow(11).int();
+
+// Set function to generate cloud mask
+function maskS2clouds(image) {
+  var qa = image.select('QA60');
+// Both flags should be set to zero, indicating clear conditions.
+  var mask = qa.bitwiseAnd(cloudBitMask).eq(0).and(
+             qa.bitwiseAnd(cirrusBitMask).eq(0));
+  return image.updateMask(mask);
+}
+
+// Apply the cloud mask with other filters to derive a mosaic within spatial and temporal context
+var cloudMasked = s2.filterBounds(aoi).map(maskS2clouds)
+.filterDate('2017-06-15', '2017-10-15');
+var min = cloudMasked.min();
+var mosaic = ee.ImageCollection(min).mosaic();
+
+// Create Custom mosaic from selected bands to visualize the cloud-minimized imagery
+// You can apply similar code to compare initial cloud-contaminated imagery, setting "s2" as the image
+Map.addLayer(mosaic, {bands: ['B4', 'B3', 'B2'],
+max: 2000}, 'custom mosaic');
+```
+
+## 2. Setting-Up and Implementing Analytics
+### a. Now that you have prepared the mosaic, proceed to select the spectral bands that are relevant for the classification. By selecting more bands, the analysis will become more computationally intensive. The bands also have differing spatial resolution (https://en.wikipedia.org/wiki/Sentinel-2), so the relative computation cost (and time) will vary with the choice of bands. In the code below, all bands of the S2A are selected, but you can tweak this by selecting fewer bands. Note that our goal is to utilize as much spectral information as possible to train the classifier algorithm to differentiate between maize and non-maize. The training points (trainpts) will be used to extract the reflectance values of the pixels from all spectral bands and this will be passed to the classifier algorithms.
+
+```js
+
+// Specify and select bands that will be used in the classification
+var bands = ['B1', 'B2', 'B3', 'B4', 'B5', 'B6', 'B7', 'B8', 'B8A', 'B9', 'B10', 'B11', 'B12'];
+
+var Image_cl = mosaic
```

Comment:
The standard naming convention for variables in JavaScript is lowerCamelCase (without underscores). Could you kindly update?

Reply:
Now resolved...didn't realize that the underscore is throwing things out of the kilter